deepcave/evaluators/ablation.py

```diff
@@ -242,10 +242,10 @@ def _ablation(
             continue_ablation, max_hp, max_hp_performance, max_hp_std
         """
         max_hp = ""
-        max_hp_difference = -np.inf
+        max_hp_difference = 0 # so that no hp beloy the default cost is chosen
 
```

Comment:
I think the reason for setting it to `-np.inf` here was that if there is no improvement, we still want to choose a HP (the one that brings the least increase in cost).